@here/harp-map-theme/resources/berlin_tilezen_base.json

```diff
@@ -1118,7 +1118,12 @@
                     "fadeNear": 0.9,
                     "fadeFar": 1,
                     "lineFadeNear": -0.75,
-                    "lineFadeFar": 1
+                    "lineFadeFar": 1,
+                    "animateExtrusion": {
+                        "interpolation": "Discrete",
+                        "zoomLevels": [14, 15],
+                        "values": [false, true]
+                    }
```

Comment:
The parameter `animateExtrusion` in `ExtrudedPolygonTechniqueParams` should be changed from `boolean` to `MaybeInterpolatedProperty<boolean>`